qwik-graphql-tailwind/src/components/pull-request-issue-tab/pull-request-issue-tab.tsx

```diff
@@ -124,7 +129,7 @@ export const PullRequestIssueTab = component$(
           )}
           <div>
             <FilterDropdown name="Sort" description="Sort by" buttonClassName="border-none text-sm">
-              {sortOption.map(({ label, value }) => (
+              {sortOption?.map(({ label, value }) => (
```

Comment:
I think this question mark is unnecessary. We always have the sortOption array.

Reply:
The reason i didn't comment on this was because when declaring the type sortOption can be optional. Not sure why.

Reply:
In PR we get the sort options from the API, so we don't always have it.